packages/twenty-front/src/pages/settings/developers/webhooks/SettingsDevelopersWebhooksNew.tsx

```diff
@@ -23,11 +24,19 @@ export const SettingsDevelopersWebhooksNew = () => {
     targetUrl: '',
     operation: '*.*',
   });
+  const [errorMessage, setErrorMessage] = useState<string | undefined>();
   const { createOneRecord: createOneWebhook } = useCreateOneRecord<Webhook>({
     objectNameSingular: CoreObjectNameSingular.Webhook,
   });
   const handleSave = async () => {
+    setErrorMessage(undefined);
+
+    if (!isURL(formValues.targetUrl)) {
+      setErrorMessage('Invalid webhook URL');
```

Comment:
@rostaklein now that we are not throwing anymore, we need to return and not actually create the webhook